openapi/paths/API@bpm@archivedFailure@flowNode@{flowNodeInstanceId}.yaml

```diff
@@ -0,0 +1,36 @@
+get:
+  tags:
+    - ArchivedFailure
+  summary: Retrieve the archived failures of an archived flownode instance by it's sourceObjectID
+  description: |
+    Returns the most recent failures for the given archived flownode sourceObjectID
+  operationId: getArchivedBPMFailuresByFlowNodeInstanceId
+  parameters:
+    - description: The original identifier (sourceObjectID) of the flownode instance from which to retrieve the failures
+      in: path
+      name: flowNodeInstanceId
+      required: true
+      schema:
+        type: string
+        maxLength: 250
+        pattern: '^[A-Za-z0-9\_\-\.]{0,250}$'
```

Comment:
Not sure A-Zaz is part of the pattern of a flownode id

Reply:
```suggestion
        pattern: '^[0-9\_\-\.]{0,250}$'
```